commit befa265294fd9ff679dece7874c1de50fea97b25
Author: pradeep <pradeep@arrayfire.com>
Date:   Wed Jun 13 15:10:23 2018 +0530

    Remove experimental glm/transform header
    
    Starting with glm-0.9.9, experimental headers can only
    be included if GLM_ENABLE_EXPERIMENTAL is defined before
    the inclusion of the header.

diff --git a/src/backend/opengl/font_impl.cpp b/src/backend/opengl/font_impl.cpp
index 05b20f3..d1ad889 100644
--- a/src/backend/opengl/font_impl.cpp
+++ b/src/backend/opengl/font_impl.cpp
@@ -20,7 +20,6 @@
 
 #include <glm/glm.hpp>
 #include <glm/gtc/matrix_transform.hpp>
-#include <glm/gtx/transform.hpp>
 
 #include <ft2build.h>
 #include FT_FREETYPE_H
